askama_parser/src/node.rs

```diff
@@ -626,6 +632,31 @@ impl<'a> Match<'a> {
     }
 }
 
+fn check_end_name<'a>(
```

Comment:
Moving this should be in a separate commit from making changes to it to accomodate the second use case. IMO if you already know you're going to take it out of the parent function, better to keep it at the top-level in the introducing commit.

Please keep it ordered below the highest user.

Reply:
Ok, I'll move it out directly then.